[29.09.2021 13:17:58.761] # ЗАПУСТИЛИ ПОЗИТИВНЫЙ СМОУК-ТЕСТ НА СОЗДАНИЕ ПРОПУСКА ПО ЗАЯВКЕ НА РАЗОВЫЙ ПРОПУСК
[29.09.2021 13:17:58.762] # установили нужный размер окна браузера
[29.09.2021 13:17:58.893] # заходим на страницу регистрации в VISITORCONTROL
[29.09.2021 13:17:59.169] # логинимся
[29.09.2021 13:17:59.666] # смотрим результат
[29.09.2021 13:18:02.678] # заходим в раздел "Пропуска"
[29.09.2021 13:18:02.756] # заходим на страницу создания пропуска по заявке
[29.09.2021 13:18:02.837] # Заполняем вкладку "Посетитель"
[29.09.2021 13:18:02.837] # Выбираем фамилию из выпадающего списка фамилий в заявках
[29.09.2021 13:18:03.187] # 1
[29.09.2021 13:18:03.261] # 2
[29.09.2021 13:18:03.861] # 3
[29.09.2021 13:18:03.861] # Открываем список свободных карт доступа
[29.09.2021 13:18:04.139] # Кликаем первую свободную карту
[29.09.2021 13:18:04.249] # Выбираем эту карту
[29.09.2021 13:18:07.320] # coхраняем пропуск
[29.09.2021 13:18:07.387] # обязательно здесь time.sleep(), иначе пропуск не сохраняется
[29.09.2021 13:18:16.395] # закрываем окно с пропуском
[29.09.2021 13:18:16.521] # открываем все пропуска
[29.09.2021 13:18:16.848] # выбираем только что созданный пропуск
[29.09.2021 13:18:17.783] # кликаем кнопку "Внести отметку" ("Время сдачи пропуска")
[29.09.2021 13:18:18.146] # кликаем кнопку "Ок" ("Отметка о выходе из здания внесена!")
[29.09.2021 13:18:18.276] # ЗАВЕРШИЛИ ПОЗИТИВНЫЙ СМОУК-ТЕСТ НА СОЗДАНИЕ РАЗОВОГО ПРОПУСКА ПО ЗАЯВКЕ
[29.09.2021 13:18:18.276] ****************************************************************************
[29.09.2021 13:18:18.276] # заходим в раздел "Пропуска"
[29.09.2021 13:18:18.559] # заходим на страницу создания пропуска по заявке
[29.09.2021 13:18:18.635] # Заполняем вкладку "Посетитель"
[29.09.2021 13:18:18.636] # Выбираем фамилию из выпадающего списка фамилий в заявках
[29.09.2021 13:18:19.063] # 1
[29.09.2021 13:18:19.119] # 2
[29.09.2021 13:18:19.558] # 3
[29.09.2021 13:18:19.558] # Открываем список свободных карт доступа
[29.09.2021 13:18:19.824] # Кликаем первую свободную карту
[29.09.2021 13:18:19.949] # Выбираем эту карту
[29.09.2021 13:18:23.014] # coхраняем пропуск
[29.09.2021 13:18:23.095] # обязательно здесь time.sleep(), иначе пропуск не сохраняется
[29.09.2021 13:18:32.103] # закрываем окно с пропуском
[29.09.2021 13:18:32.205] # открываем все пропуска
[29.09.2021 13:18:32.503] # выбираем только что созданный пропуск
[29.09.2021 13:18:33.355] # кликаем кнопку "Внести отметку" ("Время сдачи пропуска")
[29.09.2021 13:18:33.715] # кликаем кнопку "Ок" ("Отметка о выходе из здания внесена!")
[29.09.2021 13:18:33.842] # ЗАВЕРШИЛИ ПОЗИТИВНЫЙ СМОУК-ТЕСТ НА СОЗДАНИЕ РАЗОВОГО ПРОПУСКА ПО ЗАЯВКЕ
[29.09.2021 13:18:33.843] ****************************************************************************
[29.09.2021 13:18:33.843] # заходим в раздел "Пропуска"
[29.09.2021 13:18:34.158] # заходим на страницу создания пропуска по заявке
[29.09.2021 13:18:34.269] # Заполняем вкладку "Посетитель"
[29.09.2021 13:18:34.269] # Выбираем фамилию из выпадающего списка фамилий в заявках
[29.09.2021 13:18:34.554] # 1
[29.09.2021 13:18:34.616] # 2
[29.09.2021 13:18:34.991] # 3
[29.09.2021 13:18:34.991] # Открываем список свободных карт доступа
[29.09.2021 13:18:35.254] # Кликаем первую свободную карту
[29.09.2021 13:18:35.369] # Выбираем эту карту
[29.09.2021 13:18:38.436] # coхраняем пропуск
[29.09.2021 13:18:38.552] # обязательно здесь time.sleep(), иначе пропуск не сохраняется
[29.09.2021 13:18:47.567] # закрываем окно с пропуском
[29.09.2021 13:18:47.676] # открываем все пропуска
[29.09.2021 13:18:47.981] # выбираем только что созданный пропуск
[29.09.2021 13:18:48.885] # кликаем кнопку "Внести отметку" ("Время сдачи пропуска")
[29.09.2021 13:18:49.261] # кликаем кнопку "Ок" ("Отметка о выходе из здания внесена!")
[29.09.2021 13:18:49.383] # ЗАВЕРШИЛИ ПОЗИТИВНЫЙ СМОУК-ТЕСТ НА СОЗДАНИЕ РАЗОВОГО ПРОПУСКА ПО ЗАЯВКЕ
[29.09.2021 13:18:49.383] ****************************************************************************
[29.09.2021 13:18:49.383] # заходим в раздел "Пропуска"
[29.09.2021 13:18:49.735] # заходим на страницу создания пропуска по заявке
[29.09.2021 13:18:49.814] # Заполняем вкладку "Посетитель"
[29.09.2021 13:18:49.814] # Выбираем фамилию из выпадающего списка фамилий в заявках
[29.09.2021 13:18:50.118] # 1
[29.09.2021 13:18:50.179] # 2
[29.09.2021 13:18:50.592] # 3
[29.09.2021 13:18:50.592] # Открываем список свободных карт доступа
[29.09.2021 13:18:50.892] # Кликаем первую свободную карту
[29.09.2021 13:18:50.999] # Выбираем эту карту
[29.09.2021 13:18:54.058] # coхраняем пропуск
[29.09.2021 13:18:54.135] # обязательно здесь time.sleep(), иначе пропуск не сохраняется
[29.09.2021 13:19:03.146] # закрываем окно с пропуском
[29.09.2021 13:19:03.195] # открываем все пропуска
[29.09.2021 13:19:03.493] # выбираем только что созданный пропуск
[29.09.2021 13:19:04.471] # кликаем кнопку "Внести отметку" ("Время сдачи пропуска")
[29.09.2021 13:19:04.850] # кликаем кнопку "Ок" ("Отметка о выходе из здания внесена!")
[29.09.2021 13:19:05.073] # ЗАВЕРШИЛИ ПОЗИТИВНЫЙ СМОУК-ТЕСТ НА СОЗДАНИЕ РАЗОВОГО ПРОПУСКА ПО ЗАЯВКЕ
[29.09.2021 13:19:05.073] ****************************************************************************
[29.09.2021 13:19:05.074] # заходим в раздел "Пропуска"
[29.09.2021 13:19:05.349] # заходим на страницу создания пропуска по заявке
[29.09.2021 13:19:05.418] # Заполняем вкладку "Посетитель"
[29.09.2021 13:19:05.419] # Выбираем фамилию из выпадающего списка фамилий в заявках
[29.09.2021 13:19:05.650] # 1
[29.09.2021 13:19:05.716] # 2
[29.09.2021 13:19:06.090] # 3
[29.09.2021 13:19:06.091] # Открываем список свободных карт доступа
[29.09.2021 13:19:06.433] # Кликаем первую свободную карту
[29.09.2021 13:19:06.561] # Выбираем эту карту
[29.09.2021 13:19:09.634] # coхраняем пропуск
[29.09.2021 13:19:09.736] # обязательно здесь time.sleep(), иначе пропуск не сохраняется
[29.09.2021 13:19:18.745] # закрываем окно с пропуском
[29.09.2021 13:19:18.860] # открываем все пропуска
[29.09.2021 13:19:19.154] # выбираем только что созданный пропуск
[29.09.2021 13:19:20.237] # кликаем кнопку "Внести отметку" ("Время сдачи пропуска")
[29.09.2021 13:19:20.605] # кликаем кнопку "Ок" ("Отметка о выходе из здания внесена!")
[29.09.2021 13:19:20.783] # ЗАВЕРШИЛИ ПОЗИТИВНЫЙ СМОУК-ТЕСТ НА СОЗДАНИЕ РАЗОВОГО ПРОПУСКА ПО ЗАЯВКЕ
[29.09.2021 13:19:20.783] ****************************************************************************
[29.09.2021 13:19:20.783] # заходим в раздел "Пропуска"
[29.09.2021 13:19:21.145] # заходим на страницу создания пропуска по заявке
[29.09.2021 13:19:21.251] # Заполняем вкладку "Посетитель"
[29.09.2021 13:19:21.251] # Выбираем фамилию из выпадающего списка фамилий в заявках
[29.09.2021 13:19:21.484] # 1
[29.09.2021 13:19:21.540] # 2
[29.09.2021 13:19:21.978] # 3
[29.09.2021 13:19:21.978] # Открываем список свободных карт доступа
[29.09.2021 13:19:22.322] # Кликаем первую свободную карту
[29.09.2021 13:19:22.426] # Выбираем эту карту
[29.09.2021 13:19:25.491] # coхраняем пропуск
[29.09.2021 13:19:25.605] # обязательно здесь time.sleep(), иначе пропуск не сохраняется
[29.09.2021 13:19:34.612] # закрываем окно с пропуском
[29.09.2021 13:19:34.685] # открываем все пропуска
[29.09.2021 13:19:34.997] # выбираем только что созданный пропуск
[29.09.2021 13:19:35.934] # кликаем кнопку "Внести отметку" ("Время сдачи пропуска")
[29.09.2021 13:19:36.322] # кликаем кнопку "Ок" ("Отметка о выходе из здания внесена!")
[29.09.2021 13:19:36.438] # ЗАВЕРШИЛИ ПОЗИТИВНЫЙ СМОУК-ТЕСТ НА СОЗДАНИЕ РАЗОВОГО ПРОПУСКА ПО ЗАЯВКЕ
[29.09.2021 13:19:36.438] ****************************************************************************
[29.09.2021 13:19:36.438] # заходим в раздел "Пропуска"
[29.09.2021 13:19:36.714] # заходим на страницу создания пропуска по заявке
[29.09.2021 13:19:36.769] # Заполняем вкладку "Посетитель"
[29.09.2021 13:19:36.769] # Выбираем фамилию из выпадающего списка фамилий в заявках
[29.09.2021 13:19:36.974] # 1
[29.09.2021 13:19:37.053] # 2
[29.09.2021 13:19:37.455] # 3
[29.09.2021 13:19:37.456] # Открываем список свободных карт доступа
[29.09.2021 13:19:37.817] # Кликаем первую свободную карту
[29.09.2021 13:19:37.929] # Выбираем эту карту
[29.09.2021 13:19:40.979] # coхраняем пропуск
[29.09.2021 13:19:41.074] # обязательно здесь time.sleep(), иначе пропуск не сохраняется
[29.09.2021 13:19:50.083] # закрываем окно с пропуском
[29.09.2021 13:19:50.178] # открываем все пропуска
[29.09.2021 13:19:50.472] # выбираем только что созданный пропуск
[29.09.2021 13:19:51.531] # кликаем кнопку "Внести отметку" ("Время сдачи пропуска")
[29.09.2021 13:19:51.926] # кликаем кнопку "Ок" ("Отметка о выходе из здания внесена!")
[29.09.2021 13:19:52.076] # ЗАВЕРШИЛИ ПОЗИТИВНЫЙ СМОУК-ТЕСТ НА СОЗДАНИЕ РАЗОВОГО ПРОПУСКА ПО ЗАЯВКЕ
[29.09.2021 13:19:52.076] ****************************************************************************
[29.09.2021 13:19:52.076] # заходим в раздел "Пропуска"
[29.09.2021 13:19:52.440] # заходим на страницу создания пропуска по заявке
[29.09.2021 13:19:53.840] An error has been caught in function 'pytest_pyfunc_call', process 'MainProcess' (9520), thread 'MainThread' (15320):
Traceback (most recent call last):

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
           |         |     -> {'__name__': '__main__', '__doc__': None, '__package__': '', '__loader__': <zipimporter object "C:\Users\zhuravlev\AppData\Lo...
           |         -> <code object <module> at 0x000001D6C70FE450, file "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\Scripts\pytest.e...
           -> <function _run_code at 0x000001D6C6D9E9D0>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
         |     -> {'__name__': '__main__', '__doc__': None, '__package__': '', '__loader__': <zipimporter object "C:\Users\zhuravlev\AppData\Lo...
         -> <code object <module> at 0x000001D6C70FE450, file "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\Scripts\pytest.e...

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\Scripts\pytest.exe\__main__.py", line 7, in <module>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\config\__init__.py", line 185, in console_main
    code = main()
           -> <function main at 0x000001D6C76A4C10>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\config\__init__.py", line 162, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
         |     |                |      |    -> <_HookCaller 'pytest_cmdline_main'>
         |     |                |      -> <pluggy._hooks._HookRelay object at 0x000001D6C70F99D0>
         |     |                -> <_pytest.config.Config object at 0x000001D6C90DD4C0>
         |     -> <enum 'ExitCode'>
         -> typing.Union

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
           |    |         |    |     |    |                |       -> True
           |    |         |    |     |    |                -> {'config': <_pytest.config.Config object at 0x000001D6C90DD4C0>}
           |    |         |    |     |    -> <function _HookCaller.get_hookimpls at 0x000001D6C75CE5E0>
           |    |         |    |     -> <_HookCaller 'pytest_cmdline_main'>
           |    |         |    -> 'pytest_cmdline_main'
           |    |         -> <_HookCaller 'pytest_cmdline_main'>
           |    -> <bound method PluginManager._hookexec of <_pytest.config.PytestPluginManager object at 0x000001D6C907D4C0>>
           -> <_HookCaller 'pytest_cmdline_main'>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           |    |               |          |        |       -> True
           |    |               |          |        -> {'config': <_pytest.config.Config object at 0x000001D6C90DD4C0>}
           |    |               |          -> [<HookImpl plugin_name='main', plugin=<module '_pytest.main' from 'C:\\Users\\zhuravlev\\AppData\\Local\\Programs\\Python\\Py...
           |    |               -> 'pytest_cmdline_main'
           |    -> <function _multicall at 0x000001D6C75C18B0>
           -> <_pytest.config.PytestPluginManager object at 0x000001D6C907D4C0>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
          |         |         -> [<_pytest.config.Config object at 0x000001D6C90DD4C0>]
          |         -> <function pytest_cmdline_main at 0x000001D6C775EE50>
          -> <HookImpl plugin_name='main', plugin=<module '_pytest.main' from 'C:\\Users\\zhuravlev\\AppData\\Local\\Programs\\Python\\Pyt...

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\main.py", line 316, in pytest_cmdline_main
    return wrap_session(config, _main)
           |            |       -> <function _main at 0x000001D6C775EEE0>
           |            -> <_pytest.config.Config object at 0x000001D6C90DD4C0>
           -> <function wrap_session at 0x000001D6C775EDC0>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\main.py", line 269, in wrap_session
    session.exitstatus = doit(config, session) or 0
    |       |            |    |       -> <Session Кастомные тесты exitstatus=<ExitCode.OK: 0> testsfailed=0 testscollected=1>
    |       |            |    -> <_pytest.config.Config object at 0x000001D6C90DD4C0>
    |       |            -> <function _main at 0x000001D6C775EEE0>
    |       -> <ExitCode.OK: 0>
    -> <Session Кастомные тесты exitstatus=<ExitCode.OK: 0> testsfailed=0 testscollected=1>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\main.py", line 323, in _main
    config.hook.pytest_runtestloop(session=session)
    |      |    |                          -> <Session Кастомные тесты exitstatus=<ExitCode.OK: 0> testsfailed=0 testscollected=1>
    |      |    -> <_HookCaller 'pytest_runtestloop'>
    |      -> <pluggy._hooks._HookRelay object at 0x000001D6C70F99D0>
    -> <_pytest.config.Config object at 0x000001D6C90DD4C0>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
           |    |         |    |     |    |                |       -> True
           |    |         |    |     |    |                -> {'session': <Session Кастомные тесты exitstatus=<ExitCode.OK: 0> testsfailed=0 testscollected=1>}
           |    |         |    |     |    -> <function _HookCaller.get_hookimpls at 0x000001D6C75CE5E0>
           |    |         |    |     -> <_HookCaller 'pytest_runtestloop'>
           |    |         |    -> 'pytest_runtestloop'
           |    |         -> <_HookCaller 'pytest_runtestloop'>
           |    -> <bound method PluginManager._hookexec of <_pytest.config.PytestPluginManager object at 0x000001D6C907D4C0>>
           -> <_HookCaller 'pytest_runtestloop'>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           |    |               |          |        |       -> True
           |    |               |          |        -> {'session': <Session Кастомные тесты exitstatus=<ExitCode.OK: 0> testsfailed=0 testscollected=1>}
           |    |               |          -> [<HookImpl plugin_name='main', plugin=<module '_pytest.main' from 'C:\\Users\\zhuravlev\\AppData\\Local\\Programs\\Python\\Py...
           |    |               -> 'pytest_runtestloop'
           |    -> <function _multicall at 0x000001D6C75C18B0>
           -> <_pytest.config.PytestPluginManager object at 0x000001D6C907D4C0>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
          |         |         -> [<Session Кастомные тесты exitstatus=<ExitCode.OK: 0> testsfailed=0 testscollected=1>]
          |         -> <function pytest_runtestloop at 0x000001D6C7763040>
          -> <HookImpl plugin_name='main', plugin=<module '_pytest.main' from 'C:\\Users\\zhuravlev\\AppData\\Local\\Programs\\Python\\Pyt...

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\main.py", line 348, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
    |    |                                        |              -> None
    |    |                                        -> <Function test_positive_smouk_temporary_pass>
    |    -> <member 'config' of 'Node' objects>
    -> <Function test_positive_smouk_temporary_pass>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
           |    |         |    |     |    |                |       -> True
           |    |         |    |     |    |                -> {'item': <Function test_positive_smouk_temporary_pass>, 'nextitem': None}
           |    |         |    |     |    -> <function _HookCaller.get_hookimpls at 0x000001D6C75CE5E0>
           |    |         |    |     -> <_HookCaller 'pytest_runtest_protocol'>
           |    |         |    -> 'pytest_runtest_protocol'
           |    |         -> <_HookCaller 'pytest_runtest_protocol'>
           |    -> <bound method PluginManager._hookexec of <_pytest.config.PytestPluginManager object at 0x000001D6C907D4C0>>
           -> <_HookCaller 'pytest_runtest_protocol'>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           |    |               |          |        |       -> True
           |    |               |          |        -> {'item': <Function test_positive_smouk_temporary_pass>, 'nextitem': None}
           |    |               |          -> [<HookImpl plugin_name='runner', plugin=<module '_pytest.runner' from 'C:\\Users\\zhuravlev\\AppData\\Local\\Programs\\Python...
           |    |               -> 'pytest_runtest_protocol'
           |    -> <function _multicall at 0x000001D6C75C18B0>
           -> <_pytest.config.PytestPluginManager object at 0x000001D6C907D4C0>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
          |         |         -> [<Function test_positive_smouk_temporary_pass>, None]
          |         -> <function pytest_runtest_protocol at 0x000001D6C775B8B0>
          -> <HookImpl plugin_name='runner', plugin=<module '_pytest.runner' from 'C:\\Users\\zhuravlev\\AppData\\Local\\Programs\\Python\...

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\runner.py", line 109, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
    |               |              -> None
    |               -> <Function test_positive_smouk_temporary_pass>
    -> <function runtestprotocol at 0x000001D6C775B940>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\runner.py", line 126, in runtestprotocol
    reports.append(call_and_report(item, "call", log))
    |       |      |               |             -> True
    |       |      |               -> <Function test_positive_smouk_temporary_pass>
    |       |      -> <function call_and_report at 0x000001D6C775BD30>
    |       -> <method 'append' of 'list' objects>
    -> [<TestReport 'test_100_by_request_positive_smoke_request_one-time_pass.py::test_positive_smouk_temporary_pass' when='setup' o...

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\runner.py", line 215, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
           |                 |     |       -> {}
           |                 |     -> 'call'
           |                 -> <Function test_positive_smouk_temporary_pass>
           -> <function call_runtest_hook at 0x000001D6C775BE50>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\runner.py", line 254, in call_runtest_hook
    return CallInfo.from_call(
           |        -> <classmethod object at 0x000001D6C7756B50>
           -> <class '_pytest.runner.CallInfo'>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\runner.py", line 311, in from_call
    result: Optional[TResult] = func()
            |        |          -> <function call_runtest_hook.<locals>.<lambda> at 0x000001D6C9E64C10>
            |        -> +TResult
            -> typing.Optional

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\runner.py", line 255, in <lambda>
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
            |          |       -> {}
            |          -> <Function test_positive_smouk_temporary_pass>
            -> <_HookCaller 'pytest_runtest_call'>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
           |    |         |    |     |    |                |       -> False
           |    |         |    |     |    |                -> {'item': <Function test_positive_smouk_temporary_pass>}
           |    |         |    |     |    -> <function _HookCaller.get_hookimpls at 0x000001D6C75CE5E0>
           |    |         |    |     -> <_HookCaller 'pytest_runtest_call'>
           |    |         |    -> 'pytest_runtest_call'
           |    |         -> <_HookCaller 'pytest_runtest_call'>
           |    -> <bound method PluginManager._hookexec of <_pytest.config.PytestPluginManager object at 0x000001D6C907D4C0>>
           -> <_HookCaller 'pytest_runtest_call'>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           |    |               |          |        |       -> False
           |    |               |          |        -> {'item': <Function test_positive_smouk_temporary_pass>}
           |    |               |          -> [<HookImpl plugin_name='runner', plugin=<module '_pytest.runner' from 'C:\\Users\\zhuravlev\\AppData\\Local\\Programs\\Python...
           |    |               -> 'pytest_runtest_call'
           |    -> <function _multicall at 0x000001D6C75C18B0>
           -> <_pytest.config.PytestPluginManager object at 0x000001D6C907D4C0>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
          |         |         -> [<Function test_positive_smouk_temporary_pass>]
          |         -> <function pytest_runtest_call at 0x000001D6C775BAF0>
          -> <HookImpl plugin_name='runner', plugin=<module '_pytest.runner' from 'C:\\Users\\zhuravlev\\AppData\\Local\\Programs\\Python\...

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\runner.py", line 162, in pytest_runtest_call
    item.runtest()
    |    -> <function Function.runtest at 0x000001D6C8FADF70>
    -> <Function test_positive_smouk_temporary_pass>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\python.py", line 1641, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
    |    |                                   -> <Function test_positive_smouk_temporary_pass>
    |    -> <property object at 0x000001D6C76C9400>
    -> <Function test_positive_smouk_temporary_pass>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
           |    |         |    |     |    |                |       -> True
           |    |         |    |     |    |                -> {'pyfuncitem': <Function test_positive_smouk_temporary_pass>}
           |    |         |    |     |    -> <function _HookCaller.get_hookimpls at 0x000001D6C75CE5E0>
           |    |         |    |     -> <_HookCaller 'pytest_pyfunc_call'>
           |    |         |    -> 'pytest_pyfunc_call'
           |    |         -> <_HookCaller 'pytest_pyfunc_call'>
           |    -> <bound method PluginManager._hookexec of <_pytest.config.PytestPluginManager object at 0x000001D6C907D4C0>>
           -> <_HookCaller 'pytest_pyfunc_call'>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           |    |               |          |        |       -> True
           |    |               |          |        -> {'pyfuncitem': <Function test_positive_smouk_temporary_pass>}
           |    |               |          -> [<HookImpl plugin_name='python', plugin=<module '_pytest.python' from 'C:\\Users\\zhuravlev\\AppData\\Local\\Programs\\Python...
           |    |               -> 'pytest_pyfunc_call'
           |    -> <function _multicall at 0x000001D6C75C18B0>
           -> <_pytest.config.PytestPluginManager object at 0x000001D6C907D4C0>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\pluggy\_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
          |         |         -> [<Function test_positive_smouk_temporary_pass>]
          |         -> <function pytest_pyfunc_call at 0x000001D6C8F9F670>
          -> <HookImpl plugin_name='python', plugin=<module '_pytest.python' from 'C:\\Users\\zhuravlev\\AppData\\Local\\Programs\\Python\...

> File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\_pytest\python.py", line 183, in pytest_pyfunc_call
    result = testfunction(**testargs)
             |              -> {'browser': <selenium.webdriver.chrome.webdriver.WebDriver (session="ba4dff22a87b14216c0b58089fb26841")>}
             -> <function test_positive_smouk_temporary_pass at 0x000001D6C9E64940>

  File "C:\Users\zhuravlev\source\repos\Test1\Кастомные тесты\test_100_by_request_positive_smoke_request_one-time_pass.py", line 51, in test_positive_smouk_temporary_pass
    Base.element_exists_and_click(browser, S.xpath_raz_propusk_by_request)
    |    |                        |        | -> '//div/a[@title="Оформление пропуска по заявке"]'
    |    |                        |        -> <class 'selectorss.Selectorss'>
    |    |                        -> <selenium.webdriver.chrome.webdriver.WebDriver (session="ba4dff22a87b14216c0b58089fb26841")>
    |    -> <function BasePage.element_exists_and_click at 0x000001D6C9E643A0>
    -> <class 'base_page.BasePage'>

  File "C:\Users\zhuravlev\source\repos\Test1\base_page.py", line 39, in element_exists_and_click
    assert BasePage.assertElementIsPresentByXPath_Click(self, xpath), f'Элемента {xpath} на странице нет'
           |        |                                   |     -> '//div/a[@title="Оформление пропуска по заявке"]'
           |        |                                   -> <selenium.webdriver.chrome.webdriver.WebDriver (session="ba4dff22a87b14216c0b58089fb26841")>
           |        -> <function BasePage.assertElementIsPresentByXPath_Click at 0x000001D6C9BD35E0>
           -> <class 'base_page.BasePage'>

  File "C:\Users\zhuravlev\source\repos\Test1\base_page.py", line 26, in assertElementIsPresentByXPath_Click
    element = self.find_element_by_xpath(xpath).click()
              |    |                     -> '//div/a[@title="Оформление пропуска по заявке"]'
              |    -> <function WebDriver.find_element_by_xpath at 0x000001D6C989EAF0>
              -> <selenium.webdriver.chrome.webdriver.WebDriver (session="ba4dff22a87b14216c0b58089fb26841")>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
    |    |        |       -> 'clickElement'
    |    |        -> <class 'selenium.webdriver.remote.command.Command'>
    |    -> <function WebElement._execute at 0x000001D6C969B280>
    -> <selenium.webdriver.remote.webelement.WebElement (session="ba4dff22a87b14216c0b58089fb26841", element="3146e058-7305-40fc-aee...

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
           |    |       |       |        -> {'id': '3146e058-7305-40fc-aee5-6b1225c91e01', 'sessionId': 'ba4dff22a87b14216c0b58089fb26841'}
           |    |       |       -> 'clickElement'
           |    |       -> <function WebDriver.execute at 0x000001D6C989E820>
           |    -> <selenium.webdriver.chrome.webdriver.WebDriver (session="ba4dff22a87b14216c0b58089fb26841")>
           -> <selenium.webdriver.remote.webelement.WebElement (session="ba4dff22a87b14216c0b58089fb26841", element="3146e058-7305-40fc-aee...

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
    |    |             |              -> {'status': 400, 'value': '{"value":{"error":"element click intercepted","message":"element click intercepted: Element \\u003C...
    |    |             -> <function ErrorHandler.check_response at 0x000001D6C988E790>
    |    -> <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x000001D6C9E5D8E0>
    -> <selenium.webdriver.chrome.webdriver.WebDriver (session="ba4dff22a87b14216c0b58089fb26841")>

  File "C:\Users\zhuravlev\AppData\Local\Programs\Python\Python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
          |               |        |       -> None
          |               |        -> None
          |               -> 'element click intercepted: Element <a class="menuItem" data-controller="VisitStandard" data-menu-item="PassByRequest" data-m...
          -> <class 'selenium.common.exceptions.ElementClickInterceptedException'>

selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <a class="menuItem" data-controller="VisitStandard" data-menu-item="PassByRequest" data-menu-item-id="2" href="/VisitStandard/CreateByRequest/PassByRequest?itemId=PassByRequest" title="Оформление пропуска по заявке">...</a> is not clickable at point (106, 536). Other element would receive the click: <div id="mhService1" class="collapsible-header menu-block-head  active">...</div>
  (Session info: chrome=94.0.4606.61)

